Echo.Process.Redis/RedisClusterImpl.cs

```diff
@@ -24,6 +24,14 @@ class RedisClusterImpl : ICluster
         Map<string, Subject<RedisValue>> subscriptions = Map<string, Subject<RedisValue>>();
         object subsync = new object();
 
+        public readonly JsonSerializerSettings JsonSerializerSettings =
+            new JsonSerializerSettings
+            {
+                TypeNameHandling = TypeNameHandling.All,
```

Comment:
I do not think this is a good idea. I would like to avoid this. I will call you tomorrow so we talk about this.